quaireaux/string/src/utils.cairo

```diff
@@ -0,0 +1,12 @@
+// These traits are intended to be temporary until similar traits are in the Cairo standard library,
+// thus their placement in utils.
+
+trait RandomAccessTrait<T, U> {
+    fn get(self: @T, index: usize) -> Option<Box<@U>>;
+    fn at(self: @T, index: usize) -> @U;
+}
```

Comment:
We now have the index trait. It's implemented for vec  https://github.com/keep-starknet-strange/alexandria/blob/fc1de0371d7ee0e9fa61c4c156df737a2adbdb04/alexandria/data_structures/src/vec.cairo#L118-L123